Palaso/Settings/CrossPlatformSettingsProvider.cs

```diff
@@ -14,17 +14,27 @@ namespace Palaso.Settings
 	/// </summary>
 	public class CrossPlatformSettingsProvider : SettingsProvider, IApplicationSettingsProvider
 	{
+		//Public only for unit testing. I can't get InternalsVisibleTo to work, possibly because of strong naming.
+		public const string UserConfigFileName = "user.config";
```

Comment:
Can't we make this (and the other members that are currently public for unit test purposes) protected and then derive a class (in the unit tests) that exposes the members we want to test?